partiql-plan/src/main/kotlin/org/partiql/plan/v1/rel/RelExcludeStep.kt

```diff
@@ -0,0 +1,40 @@
+package org.partiql.plan.v1.rel
+
+/**
+ * TODO DOCUMENTATION
+ */
+public interface RelExcludeStep : Rel {
```

Comment:
don't think the `ExcludeStep` needs to implement `Rel`

Reply:
Correct, bad paste